user_docs/en/userGuide.t2t

```diff
@@ -66,9 +66,10 @@ For further details, you can [view the full license. https://www.gnu.org/license
 For details regarding exceptions, access the license document from the NVDA menu under the "help" section.
 
 + System Requirements +[SystemRequirements]
-- Operating Systems: all 32-bit and 64-bit editions of Windows 7, Windows 8, Windows 8.1, Windows 10, and all Server Operating Systems starting from Windows Server 2008 R2.
+- Operating Systems: all 32-bit and 64-bit editions of Windows 7, Windows 8, Windows 8.1, Windows 10, Windows 11 (64-bit only), and all Server Operating Systems starting from Windows Server 2008 R2.
  - For Windows 7, NVDA requires Service Pack 1 or higher.
  - For Windows Server 2008 R2, NVDA requires Service Pack 1 or higher.
+ - Windows 11 does not support 32-bit systems but will run 32-bit programs such as NVDA.

```

Comment:
Do NVDA users need to know this, considering we have said that Windows 11 is supported? I don't think the NVDA user guide is a good source of truth for Windows 11 functionality.